README.md

```diff
@@ -84,7 +85,7 @@ docker exec -it kind-control-plane crictl images
 
 ### Deploy in a Cluster
 
-Update chart values file `yourvaluesfile.yaml` that you have previously created.
+Update a chart values file called `yourvaluesfile.yaml` that you have previously created.
```

Comment:
use `the` instead of `a`

Reply:
Added this as per your previous comment.
![image](https://user-images.githubusercontent.com/105046156/210770390-a9471c60-93ea-4ecd-8ac3-6ec898626c91.png)

Reply:
updated